app/controllers/api/v1/ite_client/records_controller.rb

```
@@ -0,0 +1,68 @@
+        end
+
+        def demographics
+          client = ::IteClient::Record.includes(
+            :demographic_profiles,
+            :addresses,
+            :phone_numbers,
+            :medicaid_ids
+          ).find(params[:id])
+
+          render_json_as_html({
+            client: client.attributes,
```

RuboCop (1.75.5)

[convention] Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

(Layout/FirstHashElementIndentation)